templates/docs/examples/patterns/image/container/aspect-ratio/16-9.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image Container / Aspect Ratio / 16/9{% endblock %}
+
+{% block standalone_css %}patterns_image{% endblock %}
+
+{% block content %}
+<div class="p-image-container--16-9 u-align--center u-vertically-center">
```

Comment:
Utilities shouldn't be required part of a component. They should only be needed if you need to modify the default behaviour that was not specified by the component itself.

If the image is supposed to be horizontally and vertically centered in the container by default, that's what the `p-image-container--` should do itself ideally.

Reply:
Revised (see [this comment](https://github.com/canonical/vanilla-framework/pull/5097#discussion_r1618861457)) and added note to documentation about centering behavior & how to change image position.